site/content/en/docs/installation/_index.md

```diff
@@ -35,7 +35,7 @@ The webhook server in kueue uses an internal cert management for provisioning ce
 To install a released version of Kueue in your cluster, run the following command:
 
 ```shell
-VERSION=v0.3.1
+VERSION={{ .Site.Params.version }}
```

Comment:
This doesn't seem to work well.

https://deploy-preview-853--kubernetes-sigs-kueue.netlify.app/docs/installation/#install-a-released-version

Reply:
Oh, great!